.flowconfig

```diff
@@ -15,6 +15,6 @@ emoji=true
 module.file_ext=.fjs
 
 module.system.node.resolve_dirname=node_modules
-module.system.node.resolve_dirname=./src/main/resources/lib
+module.system.node.resolve_dirname=./core/src/main/resources/lib
```

Comment:
This fixes the majority of Flow type errors - it looks like the directory structure was refactored in the past but this was not updated at the same time.